roles/ceph-nfs/tasks/pre_requisite_non_container.yml

```diff
@@ -1,120 +1,13 @@
 ---
-- name: add nfs-ganesha stable repository
-  yum_repository:
-    name: nfs_ganesha_stable
-    description: nfs-ganesha stable repo
-    gpgcheck: yes
-    state: present
-    gpgkey: "{{ ceph_stable_key }}"
-    baseurl: "{{ ceph_mirror }}/nfs-ganesha/rpm-{{ nfs_ganesha_stable_branch }}/{{ ceph_release }}/$basearch"
+- name: include red hat based system related tasks
+  include_tasks: pre_requisite_non_container_for_red_hat_based_systems.yml
```

Comment:
pre_requisite_non_container_redhat.yml ?

Reply:
adding an underscore between "red" and "hat"